src/azure-cli/azure/cli/command_modules/acs/agentpool_decorator.py

```diff
@@ -390,6 +391,33 @@ def _get_crg_id(self) -> Union[str, None]:
             crg_id = raw_value
         return crg_id
 
+    def get_message_of_the_day(self) -> Union[str, None]:
+        """Obtain the value of message_of_the_day.
+
+        :return: string or None
+        """
+        # read the original value passed by the command
+        message_of_the_day = None
+        message_of_the_day_file_path = self.raw_param.get("message_of_the_day")
+
+        if message_of_the_day_file_path:
+            if not os.path.isfile(message_of_the_day_file_path):
+                raise InvalidArgumentValueError(
+                    f"{message_of_the_day_file_path} is not valid file, or not accessible."
+                )
+            message_of_the_day = read_file_content(
+                message_of_the_day_file_path)
+            message_of_the_day = base64.b64encode(
+                bytes(message_of_the_day, 'ascii')).decode('ascii')
+
+        # try to read the property value corresponding to the parameter from the `mc` object
+        if self.agentpool and self.agentpool.message_of_the_day is not None:
```

Comment:
```suggestion
        if self.agentpool and hasattr(self.agentpool, "message_of_the_day") and self.agentpool.message_of_the_day is not None:
```
should add a hasattr check here, but do mind if the length of the line exceeds the limit set by linter, you need to wrap it

>       if self.agentpool and self.agentpool.message_of_the_day is not None:
E       AttributeError: 'ManagedClusterAgentPoolProfile' object has no attribute 'message_of_the_day'

Reply:
Thanks Fuming, applied the change